src/PrestaShopBundle/Controller/Admin/Sell/Order/CartController.php

```diff
@@ -139,8 +206,52 @@ public function createAction(Request $request): JsonResponse
     }
 
     /**
-     * Changes the cart address information
+     * Exports carts
+     *
+     * @param CartFilters $filters
      *
+     * @return CsvResponse
+     */
+    public function exportAction(CartFilters $filters)
+    {
+        $cartGridFactory = $this->get('prestashop.core.grid.factory.cart');
```

Comment:
```suggestion
        $filters = new CartFilters(['limit' => null] + $filters->all());
        $cartGridFactory = $this->get('prestashop.core.grid.factory.cart');
```
To avoid the same bug as the one fixed by this PR https://github.com/PrestaShop/PrestaShop/pull/16742

Reply:
Done